### ISSUE TITLE
Iconics.java;54 - GenericsUtil.getFields() causes App Not Responding (ANR)

### ISSUE THREAD

**Issue by caltseng** [mikepenz/Android-Iconics#473]
We are using ANRWatchdog and this line of code has been reported multiple times to cause ANR's. Is this a known issue? 

com.mikepenz.iconics.utils.GenericsUtil;resolveRClass;GenericsUtil.java;37
com.mikepenz.iconics.utils.GenericsUtil;getFields;GenericsUtil.java;21
com.mikepenz.iconics.Iconics;init;Iconics.java;54

## Details
- Used library version - 2.9.5
- Used support library version - 28.0.0
- Used gradle build tools version - 3.3
- Used tooling / Android Studio version - 3.5

```
Iconics.init(mApplication);
Iconics.registerFont(new IconFont());
```

**Comment by mikepenz** [comment]
The way the fonts are searched goes partially via reflection. you can optimize by manually registering the fonts and afterwards marking the init to be done like this: 
https://github.com/mikepenz/Android-Iconics/blob/develop/app/src/main/java/com/mikepenz/iconics/sample/CustomApplication.kt#L42 (uncomment that line) 

This might not yet be available in v2.9.5 though

**Comment by mikepenz** [comment]
Please update and use the mentioned approach to fix this :)
